tests/lib/c_lib/Kconfig

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2022 Meta
+
+mainmenu "Strerror test parameters"
+
+source "Kconfig.zephyr"
+
+config MINIMAL_LIBC_DISABLE_STRING_ERROR_TABLE
+    depends on MINIMAL_LIBC
```

Comment:
Use TAB for indentations in Kconfig

Reply:
I wonder if this is a vs code side effect

Reply:
would be good to install editorconfig plugin in the VS Code so that it automatically follows the repo-wide style:
https://github.com/zephyrproject-rtos/zephyr/blob/872f7a9dc4d0dc713cad25b1c8a5528becc30318/.editorconfig#L89-L92

Reply:
@stephanosio Done!